.travis.yml

```diff
@@ -1,56 +1,72 @@
 dist: trusty
+php: 7.3
 
 before_install:
     # turn off XDebug to speed up
     - phpenv config-rm xdebug.ini || return 0
 
+install:
+    - composer install
+
 jobs:
-    fast_finish: true
     include:
-        # 1. build Docker image
+        # 1. Testing & static analysis
         -
-            stage: build
-            if: NOT type = cron
-            name: Build docker image
+            stage: test
+            name: ECS
             script:
-                - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-                - docker pull pehapkari/pehapkari.cz || true
-                - docker build --cache-from pehapkari/pehapkari.cz -t pehapkari/pehapkari.cz:$TRAVIS_COMMIT --target production -f Dockerfile .
-                - docker push pehapkari/pehapkari.cz
+                - composer check-cs
 
-        # 2. using built Docker image from previous step
         -
-            stage: test
-            name: Static Analysis
+            name: PHPStan
             script:
-                - docker run --rm pehapkari/pehapkari.cz:$TRAVIS_COMMIT composer check-cs
-                - docker run --rm pehapkari/pehapkari.cz:$TRAVIS_COMMIT composer phpstan
+                - composer phpstan
 
         -
             name: PHPUnit
-            script: docker run --rm pehapkari/pehapkari.cz:$TRAVIS_COMMIT php vendor/bin/phpunit
+            script:
+                - vendor/bin/phpunit
```

Comment:
This one is not static analysis and therefore should run in docker container so it has same environment as production.

Reply:
Here it's just posts testing, not really related to src

Reply:
https://github.com/pehapkari/pehapkari.cz/tree/master/statie/tests/Posts